src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+  onDelete: (id: number) => void;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo, onDelete }) => {
+  const { title, completed } = todo;
+  const [completedStatus, setCompletedStatus] = useState(completed);
+
+  return (
+    <>
+      <div
+        data-cy="Todo"
+        className={cn('todo', { completed: completedStatus })}
+      >
+        <label className="todo__status-label">
+          <input
+            data-cy="TodoStatus"
+            type="checkbox"
+            className="todo__status"
+            checked={completedStatus}
+            onChange={(event) => {
+              setCompletedStatus(event.target.checked);
+            }}
+          />
+        </label>
+
+        <span
+          data-cy="TodoTitle"
+          className="todo__title"
+        >
+          {title}
+        </span>
+
+        {/* Remove button appears only on hover */}
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)
Fix all cases if present